cli/smartling/main.go

```diff
@@ -44,31 +43,27 @@ var cmdBefore = func(c *cli.Context) error {
 		if apiKey == "" {
 			apiKey = ProjectConfig.ApiKey
 		}
-		if projectId == "" {
-			projectId = ProjectConfig.ProjectId
+		if projectID == "" {
+			projectID = ProjectConfig.ProjectID
+		}
+		if userID == "" {
+			userID = ProjectConfig.UserID
 		}
 	}
 
 	if apiKey == "" {
 		log.Fatalln("ApiKey not specified in --apikey or", configFile)
 	}
-	if projectId == "" {
-		log.Fatalln("ProjectId not specified in --projectid or", configFile)
+	if projectID == "" {
+		log.Fatalln("ProjectID not specified in --projectid or", configFile)
 	}
-
-	var sc *smartling.Client
-	if c.Bool("sandbox") {
```

Comment:
We don't need sandbox support for anything?

Reply:
Sandbox was an api v1 thing. There is no equivalent sandbox mode for API v2.